### DISCUSSION TITLE
Safari 17 issue with flex and grid

### DISCUSSION THREAD

**Discussion by jaimeiniesta** [tailwindlabs/tailwindcss#14386] [General]
**What version of Tailwind CSS are you using?**

3.4.10

**What build tool (or framework if it abstracts the build tool) are you using?**

"autoprefixer": "^10.4.16",
"daisyui": "^4.12.10",
"postcss": "^8.4.33",
"postcss-import": "^16.0.0"

**What version of Node.js are you using?**

20.2.0

**What browser are you using?**

Safari 17

**What operating system are you using?**

macOS

**Reproduction URL**

https://play.tailwindcss.com/uzqrEtPtKX

That link was found in this other guy's question in SO
https://stackoverflow.com/questions/78335449/safari-strange-issue-with-flex-and-column-and-grids

**Describe your issue**

Some elements have broken layout and appear out of place, as can be seen in the screenshots of this issue:

https://github.com/saadeghi/daisyui/discussions/3205

This issue has been fixed in Safari 18.0 beta so maybe it's not worth looking at it, but I thought you might want to be aware of it for backwards compatibility.

Thanks!

**Comment by andrienko** [comment]
I resorted to using flexes for now.

In most cases people use grids they can be replaced with either a flexbox or a table, anyway.
My reasoning was something like that: we can't force users to use chrome, and chrome still uses safari/webkit to render on iOS anyway. And safari only updates together with iOS/MacOS. So users of older devices (say, a user with an old iPad that works perfectly but won't receive updates - it is a more than likely to happen, as our bugtracker has shown) are out of luck. So we got to support that, and test stuff.

_Perhaps it's related to the fact that TW does grids somethling like `repeat(4, 1fr)`... But I just did not want to dig into it._

Safari is the new IE xD
